### PR TITLE
AO3-3883 import date time not date

### PR DESCRIPTION
**Pull request by ariana-paris**
https://otwarchive.atlassian.net/browse/AO3-3883
https://code.google.com/p/otwarchive/issues/detail?id=3891

This uses a Datetime instead of just a Date when importing works, so that imported works will have a revised_at date that is the date and time of the latest chapter and will therefore sort correctly with works created on the same day.

### CODE REVIEW

**Comment by sarken** [comment]
Alas, this needs to be merged with master.

**Comment by sarken** [comment]
Once the merge conflict is fixed, I think this is ready to go -- I'm off to bed, so self-merge away if you like!

File: app/models/external_work.rb [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
-  validates_length_of :title, :maximum => ArchiveConfig.TITLE_MAX, 
-    :too_long=> ts("must be less than %{max} characters long.", :max => ArchiveConfig.TITLE_MAX)
-    
-  validates_length_of :summary, :allow_blank => true, :maximum => ArchiveConfig.SUMMARY_MAX, 
-    :too_long => ts("must be less than %{max} characters long.", :max => ArchiveConfig.SUMMARY_MAX)
-      
-  validates_presence_of :author, :message => ts('^Creator can\'t be blank')
-  validates_length_of :author, :maximum => AUTHOR_LENGTH_MAX, 
-    :too_long=> ts('^Creator must be less than %{max} characters long.', :max => AUTHOR_LENGTH_MAX)
+  validates_length_of :title, minimum: ArchiveConfig.TITLE_MIN,
```
Comment (ariana-paris) [inline app/models/external_work.rb]: ...yeah, so not refactoring `external_work` at this point...

**Comment by sarken** [comment]
It looks like something happened and this picked up a whole lot of unrelated changes. /o\ There's no rush to sort it out, though, since we have plans for the next several deploys.

**Comment by ariana-paris** [comment]
Looks like a merge with master gone wrong. I'll update it or create a new one to get rid of that lot.

**Pull request merged**